gadgets/trace_oomkill/gadget.yaml

```
@@ -0,0 +1,35 @@
+name: oomkill
+description: trace OOM killer
+tracers:
+  oomkill:
```

```suggestion
  events:
```

When running `ig build` with `--update-metadata` it does the following:
```yaml
+ events:
+   mapName: events
+   structName: event
  oomkill:
    mapName: events
    structName: event
```
Then the build throws the following error:
```bash
Error: validating metadata file: 1 error occurred:
	* only one tracer is allowed
```